tests/providers/gcp/services/cloudstorage/cloudstorage_bucket_public_access/cloudstorage_bucket_public_access_test.py

```diff
@@ -0,0 +1,119 @@
+import re
+from unittest import mock
+
+from tests.providers.gcp.gcp_fixtures import GCP_PROJECT_ID, set_mocked_gcp_provider
+
+
+class TestCloudStorageBucketPublicAccess:
+    def test_bucket_public_access(self):
+        cloudstorage_client = mock.MagicMock()
+
+        with mock.patch(
+            "prowler.providers.common.provider.Provider.get_global_provider",
+            return_value=set_mocked_gcp_provider(),
+        ), mock.patch(
+            "prowler.providers.gcp.services.cloudstorage.cloudstorage_bucket_public_access.cloudstorage_bucket_public_access.cloudstorage_client",
+            new=cloudstorage_client,
+        ):
+            from prowler.providers.gcp.services.cloudstorage.cloudstorage_bucket_public_access.cloudstorage_bucket_public_access import (
+                cloudstorage_bucket_public_access,
+            )
+
+            cloudstorage_client.project_ids = [GCP_PROJECT_ID]
+            cloudstorage_client.region = "global"
+
+            from prowler.providers.gcp.services.cloudstorage.cloudstorage_service import (
+                Bucket,
+            )
+
+            cloudstorage_client.buckets = [
+                Bucket(
+                    name="bucket1",
+                    id="bucket1",
```

Comment:
Use different names for `name` and `id` and try to match the most possible with GCP real resources, please

Reply:
GCP resources use the same name and ID, so I'll change both to match the format of real names.